### ISSUE TITLE
[feature/netcore] Metadata Endpoint

### ISSUE THREAD

**Issue by tonysneed** [OData/WebApi#1150]
Will the .NET Core version of OData support an endpoint that returns the EF Core model metadata?  In classic OData the $metadata endpoint returned the EF conceptual model as CSDL, but EF Core does not have a way to express the model in a readable format.

**Comment by weitzhandler** [comment]
:+1:
Very compelling indeed.

**Comment by robward-ms** [comment]
@tonysneed - No changes to the metadata endpoint for AspNetCore version. I realized today this was phrased as a question but assumed it was a feature request. Do you want to use this issue to track the feature request?

**Comment by tonysneed** [comment]
@robward-ms Yes, it would be great to use this issue to track the feature request. 😄  The main reason for an expressive metadata endpoint is to aid client-side tooling and code generation.
